localstack/services/logs/logs_listener.py

```diff
@@ -1,9 +1,54 @@
 import re
-from requests.models import Request
-from localstack.utils.common import to_str
+import json
+from requests.models import Request, Response
+from localstack.utils.common import to_str, now
 from localstack.constants import APPLICATION_AMZ_JSON_1_1
 from localstack.services.generic_proxy import ProxyListener
 
+subscription_filters = []
+
+
+def handle_put_subscription_filter(data):
+    data = json.loads(data)
+    filter_name = data.get('filterName')
```

Comment:
nit: We could use `select_attributes(..)` from `utils/common.py` here, to make this a bit shorter:
```
attrs = ('filterName', 'logGroupName', 'filterPattern', 'destinationArn', 'roleArn')
entry = select_attributes(data, attrs)
entry['creationTime'] = now_utc()
subscription_filters.append(entry)
```